src/containers/ActionWrapper.js

```diff
@@ -7,7 +7,6 @@ import * as actions from "actions/ActionWrapper";
 const mapStateToProps = (state, props) => {
   return {
     redirect: state.config.redirect,
-    is_fetching: state.config.is_fetching
```

Comment:
I just want to double check that it's ok to go beyond commenting this out here and fully remove it

Reply:
The actionWrapper does not need to know whether the app is fetching any more; just the buttons need to know